src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -355,9 +365,9 @@ def _process_mash_tool(root_dir: str,
                 raise ValueError(f'Unable to locate the sketch file: {sketch_file} for genome: {data_id}')
             sketch_files.append(sketch_file)
 
-    create_import_files(root_dir, f'{kbase_collection}_{load_ver}_{SEQ_METADATA}', seq_meta)
+    create_import_files(root_dir, env, f'{kbase_collection}_{load_ver}_{SEQ_METADATA}', seq_meta)
 
-    import_dir = Path(root_dir, loader_common_names.IMPORT_DIR)
+    import_dir = Path(root_dir, loader_common_names.IMPORT_DIR, env)
     os.makedirs(import_dir, exist_ok=True)
```

Comment:
Maybe there should be common code for making  the import dir

Reply:
I was thinking `make_import_dir` would go in `loader_helpers` and be used by the function that writes the import files as well

Reply:
yup yup. I realized the same after commit. Made the update.

Reply:
:+1: